crates/fluence-app-service/src/service.rs

```diff
@@ -144,23 +143,12 @@ impl AppService {
         let base_dir = &config.service_base_dir;
         let service_dir = base_dir.join(&service_id);
         create(&service_dir)?;
-
-        let local_dir = service_dir.join(SERVICE_LOCAL_DIR_NAME);
-        create(&local_dir)?;
-
-        let tmp_dir = service_dir.join(SERVICE_TMP_DIR_NAME);
-        create(&tmp_dir)?;
-
-        let local_dir = local_dir.to_string_lossy().to_string();
-        let tmp_dir = tmp_dir.to_string_lossy().to_string();
-
-        let mut preopened_files = HashSet::new();
-        preopened_files.insert(PathBuf::from(local_dir.clone()));
-        preopened_files.insert(PathBuf::from(tmp_dir.clone()));
+        create(&service_dir.join(SERVICE_LOCAL_DIR_NAME))?;
+        create(&service_dir.join(SERVICE_TMP_DIR_NAME))?;
 
         let mapped_dirs = hashmap! {
-            String::from(SERVICE_LOCAL_DIR_NAME) => PathBuf::from(local_dir),
-            String::from(SERVICE_TMP_DIR_NAME) => PathBuf::from(tmp_dir),
+            SERVICE_LOCAL_DIR_NAME.to_string() => PathBuf::from(SERVICE_LOCAL_DIR_NAME),
+            SERVICE_TMP_DIR_NAME.to_string() => PathBuf::from(SERVICE_TMP_DIR_NAME),
```

Comment:
This looks wrong. What is the meaning here? Why are you mapping `"tmp"` to `"tmp"` and `"local"` to `"local"`?

I would expect you to do
```suggestion
            SERVICE_LOCAL_DIR_NAME.to_string() => service_dir.join(SERVICE_LOCAL_DIR_NAME),
            SERVICE_TMP_DIR_NAME.to_string() => service_dir.join(SERVICE_TMP_DIR_NAME),
```

And what about any other directories, eg `/` or `/etc`? If a user writes them to config, they need to be mapped to `service_base_dir` as well.
```
mapped_dirs = { "/will_remove_this_haha" = "/etc" }
```

Reply:
Absolute paths and paths containing .. are banned here

https://github.com/fluencelabs/marine/pull/244/files#diff-7ce8317104ac1eb163bd20a0439f963f5eca4c9189c8ca911cc6262748ac6265


And all other mapped directories moved to the service_base_dir